hexrdgui/masking/mask_manager.py

```diff
@@ -249,7 +261,7 @@ def load_masks(self, h5py_group):
             else:
                 new_mask = RegionMask.deserialize(data)
             self.masks[key] = new_mask
-            new_mask.update_masked_arrays()
+            new_mask.update_masked_arrays(self.view_mode)
```

Comment:
We probably don't need this line of code, right? The mask will get generated when the `get_` method is called.

Reply:
Yes, there were quite a few redundant `update_` calls... Those have been removed now and in relevant places the they have been replaced with `invalidate_` calls.